src/api.js

```diff
@@ -1,5 +1,6 @@
-/* eslint-disable-next-line no-unused-vars */
+/* eslint-disable no-unused-vars */
```

Comment:
It has to be disabled in pretty much all our imports because React requires that we import stuff like `React` that is never explicitly used. At least that's my understanding.

Reply:
I’m almost certain this isn’t the case, the react import is only needed if you use it for react.component and whatnot

Reply:
@bobheadxi @stanleyye Here's what happens when you don't import React:
![image](https://user-images.githubusercontent.com/9647946/43560182-245017a6-95c6-11e8-9221-ddf25907aaa8.png)

Reply:
You only need it for jsx elements (the pseudo HTML stuff), don’t think you need it here. I think you can also set it as a global

Reply:
Strange though... I swear I use stateless components at work without the react import. Might be something in webpack. I’ll check tomorrow 🤷‍♀️

Reply:
btw @bfbachmann see https://stackoverflow.com/questions/42640636/react-must-be-in-scope-when-using-jsx-react-react-in-jsx-scope